frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -88,72 +152,338 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
+  name,
+  namespace,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      variant="success"
+      className="co-alert"
+      title={t('olm~Successfully deleted all Operand Instances')}
+      isInline
+    >
+      <p>
+        <Trans t={t} ns="olm">
+          All Operand instances for Operator <strong>{{ name }}</strong> in{' '}
+          <strong>{{ namespace }}</strong> have been deleted.
+        </Trans>
+      </p>
+    </Alert>
+  );
+};
+
+const UninstallAlert: React.FC<{ errorMessage: string; name: string; namespace: string }> = ({
+  errorMessage,
+  name,
+  namespace,
+}) =>
+  errorMessage ? (
+    <OperatorUninstallErrorAlert errorMessage={errorMessage} />
+  ) : (
+    <OperatorUninstallSuccessAlert name={name} namespace={namespace} />
+  );
+
+const OperandsTable: React.FC<OperandsTableProps> = ({ operands, loaded, csvName, cancel }) => {
+  const { t } = useTranslation();
+  return (
+    <StatusBox
+      skeleton={<div className="loading-skeleton--table" />}
+      data={operands}
+      loaded={loaded}
+    >
+      <table className="pf-c-table pf-m-compact pf-m-border-rows">
+        <thead>
+          <tr>
+            <th>{t('olm~Name')}</th>
+            <th>{t('olm~Kind')}</th>
+            <th>{t('olm~Namespace')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {operands
+            .sort((a, b) => a.metadata.name.localeCompare(b.metadata.name))
+            .map((operand) => (
+              <>
+                {/* eslint-disable-next-line react/no-array-index-key */}
+                <tr key={`${operand.metadata.uid}`}>
+                  <td>
+                    <OperandLink obj={operand} csvName={csvName} onClick={cancel} />
+                  </td>
+                  <td className="co-break-word" data-test-operand-kind={operand.kind}>
+                    {operand.kind}
+                  </td>
+                  <td>
+                    {operand.metadata.namespace ? (
+                      <ResourceLink
+                        kind="Namespace"
+                        name={operand.metadata.namespace}
+                        onClick={cancel}
+                      />
+                    ) : (
+                      '-'
+                    )}
+                  </td>
+                </tr>
+              </>
+            ))}
+        </tbody>
+      </table>
+    </StatusBox>
+  );
+};
+
+const OperandErrorsList: React.FC<OperandErrorsListProps> = ({
+  operandErrors,
+  csvName,
+  cancel,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <ul className="co-operator-uninstall-alert__list">
+      {_.map(operandErrors, (operandError) => (
+        <li
+          key={operandError.operand.metadata.uid}
+          className="list-unstyled co-operator-uninstall-alert__list-item"
+        >
+          <OperandLink obj={operandError.operand} csvName={csvName} onClick={cancel} />{' '}
+          {`(${operandError.operand.kind})`}
```

Comment:
```suggestion
          {operandError.operand.kind}
```

Reply:
Technically this should get the model and use the `labelKey` if present, but we can skip that for now. (This could matter for operators with plugins that can define a translated name.)